tests/pgwrr_tests.py

```diff
@@ -55,11 +55,11 @@ def test_answer():
         assert_equal(pgwrr.proto.answer(qname='name', qclass='IN', qtype='A',
                                         qcontent='10.0.0.1', qttl='60',
                                         qid='1'), None)
-        assert_equal(output.getvalue(), 'DATA\t0\t1\tname\tIN\tA\t60\t1\t10.0.0.1\nEND\n')
```

Comment:
I'm not sure I understand why END was removed

Reply:
Because we can now have several lines in a reply, so the answer() method only generates a line and the end() method has to be called at the end (seems like it would make more sense as well).